docs/CANTEEN.md

```diff
@@ -0,0 +1,44 @@
+## Canteen API
+
+### Common structures:
+
+#### Menu
+
+```
+{
+	"id": number
+	"menu": string
+	"type": number, 0-2
+}
+```
+
+### Get menus by type
+#### Path:
+`GET  /canteen/getCanteenMenus?id={type}`
+#### Paramters:
+- type - the type of the canteen, 0-2
+#### Returns:
+```
+menu[]
+```
+Returns `400: Invalid request` if type is missing, or not in 0-2
```

Comment:
If you don't add an empty new line after a row it renders them in the same line. Please add an empty line everywhere you need the row to be split